lib/providers/root/root_combine_provider.dart

```diff
@@ -25,13 +25,14 @@ final rootCombineProvider = Provider.family<void, BuildContext>((ref, context) {
       .distinct();
   streamEvent.forEach((element) {
     if (element == ProcessingState.completed) {
-      _handleAudioCompletion(ref);
+      _handleAudioCompletion(context, ref);
       _handleUserNotSignedIn(ref, context);
```

Comment:
It's good practice to keep the params in the same order. 

So like this:
` _handleAudioCompletion(context, ref);
      _handleUserNotSignedIn(context, ref);`